packages/components/src/components/dialog/dialog.stories.ts

```diff
@@ -57,271 +66,61 @@ export const Default = {
-        <sd-dialog id="timed" headline="Timed Dialog" no-close-button>
-          <div id="countdown">Closable in 5 seconds...</div>
-          <sd-button slot="footer" class="w-full" id="timed-close-button" disabled>Close</sd-button>
-        </sd-dialog>
-
-        <script>
-          // Prevent closing the dialog for a certain amount of time
-          const openTimedDialogButton = document.querySelector('#open-timed-dialog');
-          const countdownElement = document.querySelector('#countdown');
-          const timedCloseButton = document.querySelector('#timed-close-button');
-
-          const timedDialog = document.querySelector('#timed');
-          let canCloseTimedDialog = false;
-
-          timedCloseButton.addEventListener('click', () => timedDialog.hide());
-
-          openTimedDialogButton.addEventListener('click', () => {
-            timedCloseButton.disabled = true;
-            timedDialog.show();
-            canCloseTimedDialog = false;
-            let counter = 5;
-            countdownElement.textContent = 'Closable in ' + counter + ' seconds...';
-
-            const interval = setInterval(() => {
-              counter--;
-              countdownElement.textContent = 'Closable in ' + counter + ' seconds...';
-              if (counter <= 0) {
-                clearInterval(interval);
-                canCloseTimedDialog = true;
-                timedCloseButton.disabled = false;
-                countdownElement.textContent = 'You can now close the dialog.';
-              }
-            }, 1000);
-          });
-
-          timedDialog.addEventListener('sd-request-close', event => {
-            if (!canCloseTimedDialog) {
-              event.preventDefault();
-            }
-          });
-        </script>
-      </div>
-    `;
-  }
-};
-
-/**
- * sd-dialog is fully accessibile via keyboard.
+ * However, you should always include a close button to comply with ARIA principles. If the close button is omitted, the close action has to be applied to one button in the footer slot.
  */
 
-export const Mouseless = {
-  render: (args: any) => {
-    return html`<div class="mouseless" style="height: 40vh;">${generateTemplate({ args })}</div>`;
-  },
-
-  play: async ({ canvasElement }: { canvasElement: HTMLUnknownElement }) => {
-    const el = canvasElement.querySelector('.mouseless sd-dialog');
-    await waitUntil(() => el?.shadowRoot?.querySelector('sd-button'));
-
-    el?.shadowRoot?.querySelector<HTMLElement>('sd-button')!.focus();
-  }
+export const NoCloseButton = {
+  name: 'No Close Button',
+  render: () => html`
+    <div class="h-[40vh]">
+      <sd-dialog id="dialog" headline="Headline" open no-close-button>
+        <p class="sd-paragraph">
+          Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy eirmod tempor invidunt ut labore et
+          dolore magna aliquyam erat, sed diam voluptua. At vero eos et accusam.
+        </p>
+        <sd-button slot="footer">Button</sd-button>
```

Comment:
@smfonseca @coraliefeil  It seems a little strange that we suggest our users to make the button a close button to follow ARIA principles, but our example does not do that. I noticed that originally it was a close button, did we decide to move away from it on purpose? --**specifically for the no close button story**
![image](https://github.com/user-attachments/assets/e32b4130-f952-4cb8-81ae-b4cc3bf1542f)

Reply:
Good point @Vahid1919, thank you. 
@coraliefeil can you please update in figma? I've updated storybook.

Reply:
Nope. Would switch it back to just "Button" ... it’s about the action button closing the dialog ... could be sth like "Accept Terms and Conditions" or the like ... "Close" will never appear like that. It’s about a message the user has to accept or confirm so the overlay blocking the interface can close.

Reply:
@coraliefeil @smfonseca I see. I would be fine with keeping it as "Button" or "Accept Terms and Conditions". The main part is maintaining the closing behavior.

We should perhaps me more precise in our documentation in this case. Instead of: "you should always include a close button" maybe something like: "You should always include an action button that allows users to close the dialog."

Reply:
I like your suggestion @Vahid1919. What do you think @coraliefeil? Can we update the description to something like what @Vahid1919 mentioned?

Reply:
Yes. THX

![image](https://github.com/user-attachments/assets/65bd7855-2b4f-4838-8ace-495870f9f045)

Reply:
Updated according to our sync